Forta-Bots-Challenge-2/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "forta-bots-challenge-2",
```

Comment:
While this is the second challenge, I would recommend renaming it to be more specific to what the challenge entails. "Uniswap V3 Swap Detection Bot", or something closer to your `"displayName"`.